controllers/registry.go

```diff
@@ -192,8 +193,8 @@ func (r *DPAReconciler) ReconcileRegistries(log logr.Logger) (bool, error) {
 				Namespace: bsl.Namespace,
 			},
 		}
-
-		if !dpa.BackupImages() {
+		// delete registry deployment if it exists
+		// if !dpa.BackupImages() {
```

Comment:
Why comment this?

Reply:
For further discussion on if we wanna keep existing external registry at all.

Reply:
If this line wasn't commented out, registry deployment would not delete and remain running from previous version. We want to remove registry deployment from previous version since we don't use it anymore after plugin PR.

Reply:
lets just delete things if that will be the outcome then?

Reply:
sure.